src/utils.rs

```diff
@@ -112,12 +120,15 @@ pub fn bootstrap_distribution<P: AsRef<Path>, S: AsRef<str>>(
 	let mut command = Command::new("aoscbootstrap");
 	let command = if let Some(sources_list) = sources_list {
 		command.args(["--sources-list", sources_list.as_ref().to_str().unwrap()])
-	} else if let Some(mirror) = mirror {
+	} else {
 		command
 			.args(["--branch", "stable"])
-			.args(["--mirror", mirror.as_ref()])
-	} else {
-		command.args(["--branch", "stable"])
+			.args(["-s", &format!("{}/{}", AB_DIR, "scripts/reset-repo.sh")])
+			.args(if let Some(mirror) = mirror {
+				["--mirror", mirror.as_ref()]
+			} else {
+				["", ""]
+			})
```

Comment:
You are passing empty parameters to `aoscbootstrap` instead of *not* passing them. Modify this `if let` branch so that it calls `args()` optionally.